### DISCUSSION TITLE
ETA and RTC panels - different icons

### DISCUSSION THREAD

**Discussion by Ciutnik** [devemux86/cruiser#221] [Q&A] [answered]
It's nothing important, maybe to be changed on some other occasion. At first glance, you can get confused. The watch and arrival time look the same.

<img src="https://github.com/devemux86/cruiser/assets/122647782/a28317cb-7868-45c1-b1e4-d30f2932dc1f" width="33%"/>

Maybe the icons should be changed a bit? There are several possibilities: e.g. RTC icon (Real Time Clock) it could be:

[Watch](https://fonts.google.com/icons?selected=Material%20Symbols%20Outlined%3Awatch%3AFILL%400%3Bwght%40400%3BGRAD%400%3Bopsz%4048)
![watch](https://github.com/devemux86/cruiser/assets/122647782/90a624d2-31ee-403e-8322-973507b6802e)

Or/and the icon ETA at which we will arrive at the destination could look like this:

[Acute](https://fonts.google.com/icons?selected=Material%20Symbols%20Outlined%3Aacute%3AFILL%400%3Bwght%40400%3BGRAD%400%3Bopsz%4048)
![acute](https://github.com/devemux86/cruiser/assets/122647782/f43f7999-726c-4c1b-b2ad-a635346690b1)

By the way, the ETE icon could be more telling about the remaining time:

[Hourglass](https://fonts.google.com/icons?selected=Material%20Symbols%20Outlined%3Ahourglass_empty%3AFILL%400%3Bwght%40400%3BGRAD%400%3Bopsz%4048)
![hourglass](https://github.com/devemux86/cruiser/assets/122647782/9ac9131a-61d7-4f93-afe4-7184c3aebbf8)

These are little things that I've noticed for years, but sometimes they bother me a bit 🙂

**Comment by devemux86** [comment]
@Ciutnik thanks for the suggestion!

We could improve the icons, this is easy. :slightly_smiling_face: 

Can you please search icons in [Material **Icons** collection](https://fonts.google.com/icons?icon.set=Material+Icons)?
(not symbols, as they are different)

**Reply by Ciutnik** [reply]
I found [Watch](https://fonts.google.com/icons?selected=Material%20Icons%20Outlined%3Awatch%3A) and [Hourglass](https://fonts.google.com/icons?selected=Material%20Icons%20Outlined%3Ahourglass_empty%3A). I guess that's enough.

Before, I thought icons and symbols were (almost) the same thing. I was wrong 🙄

**Comment by devemux86** [comment]
Something like in the images?
- Hourglass: remaining time
- Clock: arrival time
- Watch: current time

I also changed the speed camera icon.

<img src="https://github.com/devemux86/cruiser/assets/3484020/ff6d6a56-4cd1-4af6-ab11-62871a0848dd" width="50%"/><img src="https://github.com/devemux86/cruiser/assets/3484020/081b8208-4201-485e-90e3-66c836706059" width="50%"/>

**Reply by Ciutnik** [reply]
Perfect! 👍 It is true that the arrival time icon still looks like an ordinary watch (which is not entirely clear), but we did not find anything better. Fortunately, the other time icons are different, so don't get confused. 
Thank you so much for this change!

**Reply by devemux86** [reply]
We could use the clock for the current time.
And use a different icon for the arrival time.

These are easy changes, so if anyone finds better icons we can update them!
- https://fonts.google.com/icons?icon.set=Material+Icons

**Reply by Ciutnik** [reply]
How about using a well-known [symbol](https://fonts.google.com/icons?selected=Material%20Icons%20Outlined%3Aflight_land%3A) for the arrival time?

![flight_land](https://github.com/devemux86/cruiser/assets/122647782/ea1ffd18-8b03-4c3f-9f8b-d3053d294fb2)

**Reply by devemux86** [reply]
@Ciutnik this is very related to airplanes.

Unless we add aircraft and boat profiles in Cruiser, which is now very easy! :slightly_smiling_face:

**Comment by devemux86** [comment, marked as answer]
The new navigation icons are available in [Cruiser 3.0.25](https://play.google.com/store/apps/details?id=gr.talent.cruiser).
